### PR TITLE
Firestore panel headers

### PR DESCRIPTION
**Pull request by tjlav5**
image

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ ReactDOM.render(
     // Globally disable ripples
     ripple={false}
   >
-    <ThemeProvider options={{ background, primary }}>
+    <ThemeProvider options={{ background, primary, secondary: '#F5F5F5' }}>
```
Comment (yuchenshi) [inline src/index.tsx]: Move this also into `colors`?

Comment (yuchenshi) [reply src/index.tsx]: Also, thoughts on saving secondary for the `#476282` color? It's use in quite a few places like overview text and banners in RTDB.

File: src/components/Firestore/FirestoreLogo.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2019 Google LLC
```
[the remark this answers is not in the document]
Comment (tjlav5) [reply src/components/Firestore/FirestoreLogo.tsx]: ha! copy pasta, thx

File: src/index.tsx (hunk 1 above)
Comment (tjlav5) [reply src/index.tsx]: Ack, good call... moved to colors and also isolated this surface within panel header

**Pull request merged**